recipes/hsm/all/conanfile.py

```diff
@@ -0,0 +1,48 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+from conans.tools import Version
+import os
+
+required_conan_version = ">=1.32.1"
+
+class HsmConan(ConanFile):
+    name = "hsm"
+    license = "MIT"
+    homepage = "https://github.com/erikzenker/hsm.git"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "The hana state machine (hsm) is a finite state machine library based on the boost hana meta programming library. It follows the principles of the boost msm and boost sml libraries, but tries to reduce own complex meta programming code to a minimum."
+    topics = ("state machine", "template meta programming")
```

Comment:
```suggestion
    topics = ("state-machine", "template-meta-programming")
```

I would not use space, because they could be considered as separated words when searching.